### ISSUE TITLE
asciidoc.preview.style doesn't work in 2.7.15

### ISSUE THREAD

**Issue by wetzesa** [asciidoctor/asciidoctor-vscode#307]
I would like to show my custom pdf theme in preview mode.

My project structure is like...

|- projectRoot
|-- docFolder
|--- doc.adoc
|-- shared
|--- theme
|---- theme.yml

I copied the theme.yml also to the projectRoot or to docFolder and changed the config but hadn't any affect. I changed many times the path because the config is not complete clear how it is to set. So I am now not sure if I do something wrong or the feature isn't working.

I hope someone can help me with that very very small issue ;-)

Additionally:
I am also a bit wondering about the field descripton "...path to a CSS style sheet to use...".
Must it be a CSS file and a theme.yml is not working?
The 2nd wondering in the field description was "...to the workspace folder."
Does it only work with a workspace project or is it also working if I used the "Open folder" option in VS Code?

**Comment by danyill** [comment]
Yes, it must be a path to a CSS stylesheet. This is for the generation of the html in the preview.

If I understand correctly, you are trying to generate a PDF using the Ruby asciidoctor-pdf.

I think it should work within a workspace or folder but happy to take but reports for this.

If I read the asciidoctor-pdf README correctly what you must do to use a custom theme is:

* Set the "Use_asciidoctorpdf" option: 
   ![image](https://user-images.githubusercontent.com/674783/81489958-ed93e680-92cf-11ea-85ed-27706c296b67.png)

* Set the asciidoctor-pdf command to something like: `asciidoctor-pdf -a pdf-themesdir=themes -a pdf-theme=custom -a pdf-fontsdir=fonts`.   
   ![image](https://user-images.githubusercontent.com/674783/81490079-9c84f200-92d1-11ea-9a22-d45ecb17225f.png)
   This is an OK solution but not very portable across documents which use different themes or folder structures.

Perhaps a better alternative than setting the asciidoctor-pdf command is to leave the `Asciidoctorpdf_command` as `asciidoctor-pdf` without any arguments and set in your document header the `pdf-` attributes like for instance the following:

![image](https://user-images.githubusercontent.com/674783/81490104-ed94e600-92d1-11ea-80c9-8c7e1f0f2a05.png)

@wetzesa I think your theme should have a name with a suffix `-theme.yml`, e.g. `custom-theme.yml` according to the asciidoctor-pdf theming guide.

Then for the folder structure you showed you should be able to use the document attributes:

```asciidoc
:pdf-themesdir: ../themes
:pdf-theme: custom
```

Let us know how you go!

(I tested this by using https://github.com/mderoullers/asciidoctor-pretty-theme and opening `src/main/asciidoc/example.adoc`  which is where the above screenshot came -- that might be a good example to look at if you're having trouble getting your theme to run)

**Comment by ggrossetie** [comment]
This is essentially a documentation gap rather than a bug, so I've documented both points:

- **Using a custom PDF theme** with the `asciidoctor-pdf` engine, via the `:pdf-themesdir:` / `:pdf-theme:` header attributes (resolved from the document's directory thanks to `--base-dir`): [docs/.../export-pdf.adoc](https://github.com/asciidoctor/asciidoctor-vscode/blob/main/docs/modules/ROOT/pages/export-pdf.adoc#custom-pdf-theme)
- **The preview is HTML styled with CSS**, so a PDF theme (`theme.yml`) only affects the PDF export and never changes the preview's appearance — that's why a `theme.yml` set as the "stylesheet" had no effect: [docs/.../preview.adoc](https://github.com/asciidoctor/asciidoctor-vscode/blob/main/docs/modules/ROOT/pages/preview.adoc#styling)

(Links point to the source on `main` for now; they'll be published on docs.asciidoctor.org with the next docs release.)

Closing as documented — feel free to reopen if a custom PDF theme still doesn't apply on export.
